app/presenters/summary/sections/legal_representative_details.rb

```diff
@@ -0,0 +1,45 @@
+module Summary
+  module Sections
+    class LegalRepresentativeDetails < Sections::BaseSection
+      def name
+        :legal_representative_details
+      end
+
+      def show?
+        provider_details.present? && super
+      end
+
+      # rubocop:disable Metrics/MethodLength
+      def answers
+        [
+          Components::FreeTextAnswer.new(
+            :office_code, provider_details.office_code,
+          ),
+
+          Components::FreeTextAnswer.new(
+            :provider_email, provider_details.provider_email,
+          ),
+
+          Components::FreeTextAnswer.new(
+            :legal_rep_first_name, provider_details.legal_rep_first_name,
+          ),
+
+          Components::FreeTextAnswer.new(
+            :legal_rep_last_name, provider_details.legal_rep_last_name,
+          ),
+
+          Components::FreeTextAnswer.new(
+            :legal_rep_telephone, provider_details.legal_rep_telephone, show: true,
```

Comment:
No need for `show: true` because this is a mandatory (so always present) value. Except maybe the datastore may have some corrupt data 😅